lib/src/dbs/executor.rs

```diff
@@ -34,12 +34,14 @@ impl<'a> Executor<'a> {
 	}
 
 	fn txn(&self) -> Transaction {
-		match self.txn.as_ref() {
-			Some(txn) => txn.clone(),
-			None => unreachable!(),
-		}
+		self.txn.clone().unwrap()
```

Comment:
Unwraps on `None` can be a pain to debug. I prefer `unreachable!()` here because it clearly communicates that reaching this branch is not supposed to happen. If we want to go with a one liner, `expect` with a message explaining that this is a bug might be better.